index.html

```diff
@@ -1697,6 +1697,8 @@ <h3>
           <li>Let |now:DOMHighResTimeStamp| be the [=current high resolution
           time=].
           </li>
+          <li>Let |gamepads:sequence&lt;Gamepad?&gt;| be an empty [=list=].
```

Comment:
|gamepads| here doesn't render correctly, maybe a respec bug?

Reply:
![image](https://github.com/w3c/gamepad/assets/14099571/6f4673d7-72e7-4f74-b8a6-9100745f634e)

Reply:
This should be fixed now: https://github.com/w3c/respec/pull/4709